src/tool_writeout.c

```diff
@@ -38,6 +38,10 @@ static const struct writeoutvar variables[] = {
    CURLINFO_RESPONSE_CODE, JSON_LONG},
   {"response_code", VAR_HTTP_CODE, 0,
    CURLINFO_RESPONSE_CODE, JSON_LONG},
+  {"http_response", VAR_HTTP_RESPONSE, 0,
+   CURLINFO_RESPONSE_STRING, JSON_STRING},
```

Comment:
I don't think this alias is needed. `http_code` was the original (now deprecated) name before we changed the name to `response_code` so I think we could do with only a `response_string`.

Maybe calling it **message** or **reason** is better than **string** in that name?

Reply:
I named it `response_string` to try to convey that the entire response line is returned.

Naming it `response_reason` is much worse primarily because there's no "reason" for something that is successful (e.g. HTTP `200 OK`).

Naming it `response_message` can make people think the string returned is strictly the text part of the response. Even MDN's HTTP overview page refers to the text as the (status) message.
![image](https://user-images.githubusercontent.com/1463845/92412203-ddfea000-f0ff-11ea-97b1-a481b60132cd.png)
_Source:

Reply:
I don't think an average human thinks "string" means "all text in the line". `response_line` perhaps?